### PR TITLE
#259 | Profile Page 

### PR DESCRIPTION
**Pull request by meomancer**
The profile page is not done yet, because of there are some service that still not finished. But it already work.
Note for myself :
- user link is still not there
- list of location is still not there

### CODE REVIEW

File: django_project/core/settings/contrib.py [hunk 1]
```diff
@@ -144,6 +144,15 @@
         'extra_context': {
             'media': 'screen, projection',
         },
+    },
```
Comment (timlinux) [inline django_project/core/settings/contrib.py]: These changes should be in project not contrib. Contrib is for contributed code (written by third parties), project is for code written by us.

File: django_project/social_users/models.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import itertools
+from django.contrib.auth.models import User
+from django.contrib.gis.db import models
+
+
+class UserLink(models.Model):
+    """
+    List of user link
+    """
+    link = models.CharField(blank=True, default="", null=True,
```
Comment (timlinux) [inline django_project/social_users/models.py]: please use 'pull left' for all python code we write i.e.:

```
 link = models.CharField(blank=True, default="", null=True, max_length=200)
```


```
 link = models.CharField(
    blank=True, default="", null=True, max_length=200)
```

See https://github.com/inasafe/inasafe/wiki/Coding-Standards#code-formatting for a complete set of the coding standards we should adhere to.

File: django_project/social_users/models.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+
+class UserDetail(models.Model):
+    """
+    Extention of User
+    """
+
+    user = models.OneToOneField(User, default=1)
+    link = models.ForeignKey(UserLink, default=None, blank=True, null=True)
+    profile_picture = models.FileField(
```
Comment (timlinux) [inline django_project/social_users/models.py]: If we are allowing users to upload profile pictures, we should use django hashed storage too - see https://github.com/kartoza/projecta/blob/develop/django_project/core/settings/contrib.py#L69 and we need to have a backup strategy for uploaded media.

**Comment by timlinux** [comment]
@meomancer can you address the concerns I raised above so we can merge?

**Comment by meomancer** [comment]
@timlinux

**Comment by timlinux** [comment]
@meomancer we still need migration rules in place in order to be able to merge this request.

**Comment by timlinux** [comment]
Thanks!

**Pull request merged**